test/postgres.test.coffee

```diff
@@ -48,6 +48,11 @@ test['Postgres flavour'] =
       toString: ->
         assert.same @inst.toString(), 'INSERT INTO table (field) VALUES (1) RETURNING id'
 
+    '>> into(table).set(field, 1).with(alias, table)':
+      beforeEach: -> @inst.into('table').set('field', 1).with('alias', squel.select().from('table'))
+      toString: ->
+        assert.same @inst.toString(), 'WITH alias AS (SELECT * FROM table) INSERT INTO table (field) VALUES (1)'
```

Comment:
Please add a `toParam()` test too, and the `with` query should also use params, e.g. `select().from('table').where('a = ?', 5)`